CMakeLists.txt

```diff
@@ -197,7 +197,10 @@ endif ()
 if (RAJA_ENABLE_HIP)
   set(raja_depends
     ${raja_depends}
-    hip)
+    blt_hip)
+  set(raja_depends
+    ${raja_depends}
+    blt_hip_runtime)
```

Comment:
We should be using `blt::hip` and `blt::hip_runtime` - the only place the non-prefix targets are required is when doing the TPL exports.

Reply:
changed